app/models/email.rb

```diff
@@ -15,8 +15,6 @@ class Email < ApplicationRecord
   enum status: { pending: 0, sent: 1, failed: 2 }
   enum failure_reason: { permanent_failure: 0, retries_exhausted_failure: 1, technical_failure: 2 }
 
-  validates :address, :subject, :body, presence: true
```

Comment:
Minor: you could also do this for [DigestRunSubscriber](https://github.com/alphagov/email-alert-api/blob/master/app/models/digest_run_subscriber.rb), some columns of [SubscriberList](https://github.com/alphagov/email-alert-api/blob/master/app/models/subscriber_list.rb#L11) and some of the ones for [Message](https://github.com/alphagov/email-alert-api/blob/master/app/models/message.rb#L7), which are weirdly self-contradictory (is it blank or is it not!).

Reply:
Oh cool - will take a look at what can be done there.

Reply:
I've removed the DigestRunSubscriber ones.

SubscriberList we'd still want them as this a record that is created by a HTTP request so we'd want to surface ActiveRecord errors as a 422 response - compared to the 500 response you'd get from the PostgreSQL constraints.

Same situation about Message, though you're right the phrasing seems super unintuitive with that validation.  What it actually means is that it'll only apply the CriteriaSchemaValidator when the input is not blank, which is useful so we don't have to check it's present in the validator - but doesn't come out very logical.

Reply:
Oh, I seee. Thanks for explaining!